weblate/billing/models.py

```diff
@@ -203,10 +205,13 @@ class Meta:
     def __str__(self) -> str:
         projects = self.projects_display
         owners = self.owners.order()
+        owners_visible_names_tuples = list_to_tuples([x.get_visible_name() for x in owners])
         if projects:
             base = projects
         elif owners:
-            base = ", ".join(x.get_visible_name() for x in owners)
+            base = format_html_join_comma(
+                "{}", (x for x in owners_visible_names_tuples)
```

Comment:
I have implemented the util function i created here. We would have to create an extra variable for some cases. Is this ok with you? Currently, I have only implemented it in this file. If this looks good I would apply it elsewhere as well.